sdk/core/System.ClientModel/tests/TestFramework/Mocks/MockJsonModel.cs

```diff
@@ -1,71 +0,0 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Why is this file being deleted?

Reply:
This is not deleted, just moved to `Azure.Core.TestFramework`, for shared usage.